src/coprocessor/dag/expr/ctx.rs

```diff
@@ -145,16 +158,11 @@ impl EvalConfig {
         self
     }
 
-    pub fn set_sql_mode(&mut self, new_value: u64) -> &mut Self {
+    pub fn set_sql_mode(&mut self, new_value: SqlMode) -> &mut Self {
```

Comment:
Maybe we need a better name. For bit flags, set usually means "mask" instead of "replace" (consider set_xxx_flag and unset_xxx_flag, we are not expecting to unset all other flags when calling set_xxx_flag). However the semantic of this function here is replace.

Reply:
I think it is OK since we are setting the whole value and we don't have methods to set individual bits (i.e., if this were a method on `SqlMode` it should be `replace_` but since it is a method on `EvalConfig` and the field being set is `SqlMode`, I think `set_` is correct).